recipes-ni/pstore-save/pstore-save_1.0.bb

```diff
@@ -30,7 +30,7 @@ do_install () {
 
 inherit ptest
 
-RDEPENDS_${PN}-ptest += "${PN}"
+RDEPENDS_${PN}-ptest += "${PN} bash"
```

Comment:
The 'packagegroup-core-device-devel.bb` package group you just removed, included bash. I assume there are other package groups that ensure bash is available in the feeds?

Reply:
Bash is such a common dependency that I'm sure it does end up getting built into the core feed, but it is only explcitly declared as a part of the -restoremode packagegroup. I'll add it to -base, since that will eventually mutate into the core feed packagegroup.